tests/utils/Constraint/ImageHasSizeSource.php

```diff
@@ -42,11 +42,14 @@ public function toString(): string {
 	/**
 	 * Evaluates the constraint for the provided attachment ID.
 	 *
-	 * @param int $attachment_id Attachment ID.
+	 * @param mixed $other Attachment ID.
```

Comment:
not sure I understand why this is mixed

Reply:
Ah yes, it's in order to be compatible with the class that this class extends (`\PHPUnit\Framework\Constraint\Constraint`) which is:

```php
   /**
     * Evaluates the constraint for parameter $other. Returns true if the
     * constraint is met, false otherwise.
     *
     * This method can be overridden to implement the evaluation algorithm.
     *
     * @param mixed $other value or object to evaluate
     *
     * @codeCoverageIgnore
     */
    protected function matches($other): bool
    {
        return false;
    }
```